services/gitlab/gitlab-merge-requests.service.js

```diff
@@ -0,0 +1,359 @@
+import Joi from 'joi'
+import { optionalUrl, nonNegativeInteger } from '../validators.js'
+import { metric } from '../text-formatters.js'
+import GitLabBase from './gitlab-base.js'
+
+// check schema for the response of the GitLab API
```

Comment:
I think it would be better to use the more detailed comment from down below up here:

```suggestion
// The total number of MR is in the `x-total` field in the headers.
// https://docs.gitlab.com/ee/api/index.html#other-pagination-headers
```